EIPS/eip-4906.md

```diff
@@ -0,0 +1,63 @@
+---
+eip: 4906
+title: ERC-721/ERC-1155 Metadata Update Extension
+description: Standard interface extension for ERC-721/ERC-115 metadata update 
+author: Anders (@0xanders)
+discussions-to: https://ethereum-magicians.org/t/eip4906-erc-721-erc-1155-metadata-update-extension/8588
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-03-13
+requires: 721 or 1155
```

Comment:
```suggestion
requires: 721, 1155
```

Reply:
Thanks, the document has been modified and committed.

Reply:
Thanks, the document has been modified and committed.

Reply:
@MicahZoltu if an EIP uses one of two different EIPS, should they use the notation you suggested? I've been thinking there might be a place for a notation like `721 || 1155, 165` to signify it uses either 721 **or** 1155 (but definitely 165)

Reply:
An implementation of a standard may depend on one of two other standards, but the standard itself still has two dependencies.  This means that in order to understand the full standard, you must understand all of its dependencies (if not, some of the EIP may not make sense).

If you want to keep the dependency chains separate, you can split this EIP into two EIPs, each with a different single dependency.

Reply:
> An implementation of a standard may depend on one of two other standards, but the standard itself still has two dependencies. This means that in order to understand the full standard, you must understand all of its dependencies (if not, some of the EIP may not make sense).
> 

1. `requires: 721, 1155` has been changed to  `requires: 165, 721`
2. ERC1155 relative description has been removed.